tests/Validation/ValidationValidatorTest.php

```diff
@@ -5986,6 +5986,12 @@ public function testBeforeAndAfter()
 
         $v = new Validator($trans, ['x' => ['a' => ['v' => 'c']], 'y' => 'invalid'], ['x' => 'date', 'y' => 'date|before:x']);
         $this->assertTrue($v->fails());
+
+        $v = new Validator($trans, ['y' => '1970-01-01'], ['x' => 'nullable|date', 'y' => 'nullable|date|after:x']);
+        $this->assertTrue($v->passes());
+
+        $v = new Validator($trans, ['y' => '1970-01-01'], ['x' => 'nullable|date', 'y' => 'nullable|date|before:x']);
+        $this->assertTrue($v->passes());
```

Comment:
I don't believe was ever true, so should be removed (or just reverse the condition). Essentially every date is "after" `null`

Reply:
> I don't believe was ever true, so should be removed (or just reverse the condition). Essentially every date is "after" `null`

okay before:x test was failing before adding all of those conditions too 🫡

Reply:
But after:x was not? Why did you close?

Reply:
by mistakenly added with old branch